src/repository/todo.repository/todoRepository.ts

```diff
@@ -0,0 +1,44 @@
+import fs from "fs";
+import { ToDo } from "./types";
+
+export async function saveTodo(toDo: ToDo) {
+  const todos = await getToDos();
+
+  const todoItem = todos.length
+    ? `,${JSON.stringify(toDo).trim()}`
+    : JSON.stringify(toDo).trim();
+
+  try {
+    fs.appendFile("todos.txt", todoItem, "utf-8", (error) => {
+      if (error) return { error };
+    });
+  } catch (error) {
+    return { error };
+  }
+}
+
+export default async function getToDos(): Promise<[ToDo]> {
+  const toDos: string = await fs.promises.readFile("todos.txt", "utf-8");
```

Comment:
We can move todos from a file to the memory (just an array in the repository). Then it will be easier to save/get/delete todos.